src/main.js

```diff
@@ -39,6 +16,7 @@ async function getIpStack() {
         return await ipstackResponse.json();
     } catch (error) {
         console.error('Error fetching IP Stack:', error);
+        return "NA";
```

Comment:
Any reason to use a string instead of just `null` here?

Reply:
Oh second look, this should work correctly.